corehq/apps/export/models/deid_export.py

```diff
@@ -0,0 +1,42 @@
+import hashlib
+import uuid
+
+from django.db import models
+
+from corehq.const import ONE_DAY
+from corehq.util.quickcache import quickcache
+
+
+class DeIdMapping(models.Model):
+    domain = models.TextField(max_length=255)
+    hashed_value = models.TextField(max_length=32)
+    deid = models.UUIDField(default=uuid.uuid4)
```

Comment:
I had trouble coming up with good names for these fields, with fairly little context on export code in general. Very open to alternatives here. Maybe something closer to `hashed_item` and `deid_value`?